### PR TITLE
Change default inline object name

### PR DESCRIPTION
**Pull request by Zomzog**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Default name changed for more usable one. For #2516

@wing328
@jimschubert
@cbornet
@ackintosh
@jmini

### CODE REVIEW

**Comment by wing328** [comment]
@Zomzog Thanks for the PR. I agree InlineObject is better than what we've right now.

May I suggest we also revise the warning message as follows?
```
LOGGER.warn("Found an inline schema without the `title` attribute. Default the model name to InlineObject instead. To have better control of the model naming, define the model separately so that it can be reused through out the spec.");
```

**Comment by Zomzog** [comment]
Good idea, I've updated the warning message.

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java [hunk 1]
```diff
@@ -408,8 +408,8 @@ private void addGenerated(String name, Schema model) {
 
     private String uniqueName(String key) {
         if (key == null) {
-            key = "NULL_UNIQUE_NAME";
-            LOGGER.warn("null key found. Default to NULL_UNIQUE_NAME");
+            key = "InlineObject";
+            LOGGER.warn("Found an inline schema without the `title` attribute. Default the model name to InlineObject instead. To have better control of the model naming, define the model separately so that it can be reused through out the spec.");
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java]: Sorry please make one more minor change
`through out` => `throughout` (one word)

Comment (Zomzog) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java]: changed

**Comment by wing328** [comment]
The CI failures is not caused by this change.

**Pull request merged**